tests/core/AnchorFile.spec.ts

```diff
@@ -12,140 +13,146 @@ describe('AnchorFile', async () => {
+      await expectAsync(AnchorFile.parseAndValidate(anchorFileCompressed)).toBeRejectedWith(new SidetreeError(ErrorCode.AnchorFileMapFileHashNotString));
     });
 
-    it('should throw if batch file hash is invalid.', async () => {
+    it('should throw if map file hash is invalid.', async () => {
       const anchorFile = {
-        batchFileHash: 'InvalidHash',
+        mapFileHash: 'InvalidHash',
         didUniqueSuffixes: ['EiA-GtHEOH9IcEEoBQ9p1KCMIjTmTO8x2qXJPb20ry6C0A', 'EiA4zvhtvzTdeLAg8_Pvdtk5xJreNuIpvSpCCbtiTVc8Ow']
       };
       try {
         const anchorFileBuffer = Buffer.from(JSON.stringify(anchorFile));
         const anchorFileCompressed = await Compressor.compress(anchorFileBuffer);
 
-        await AnchorFile.parseAndValidate(anchorFileCompressed, 1);
+        await AnchorFile.parseAndValidate(anchorFileCompressed);
       } catch (error) {
-        expect(error.code).toEqual(ErrorCode.AnchorFileBatchFileHashUnsupported);
+        expect(error.code).toEqual(ErrorCode.AnchorFileMapFileHashUnsupported);
       }
     });
 
     it('should throw if DID unique suffixes is not an array.', async () => {
       const anchorFile = {
-        batchFileHash: 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA',
+        mapFileHash: 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA',
         didUniqueSuffixes: 'IncorrectType'
       };
       const anchorFileBuffer = Buffer.from(JSON.stringify(anchorFile));
       const anchorFileCompressed = await Compressor.compress(anchorFileBuffer);
 
-      await expectAsync(AnchorFile.parseAndValidate(anchorFileCompressed, 1))
+      await expectAsync(AnchorFile.parseAndValidate(anchorFileCompressed))
         .toBeRejectedWith(new SidetreeError(ErrorCode.AnchorFileDidUniqueSuffixesNotArray));
     });
 
-    it('should throw if DID unique suffixes is not an array.', async () => {
+    it('should throw if operation count exceeded limit.', async () => {
       const anchorFile = {
-        batchFileHash: 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA',
+        mapFileHash: 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA',
         didUniqueSuffixes: ['EiA-GtHEOH9IcEEoBQ9p1KCMIjTmTO8x2qXJPb20ry6C0A', 'EiA4zvhtvzTdeLAg8_Pvdtk5xJreNuIpvSpCCbtiTVc8Ow']
       };
       const anchorFileBuffer = Buffer.from(JSON.stringify(anchorFile));
       const anchorFileCompressed = await Compressor.compress(anchorFileBuffer);
 
-      await expectAsync(AnchorFile.parseAndValidate(anchorFileCompressed, 1))
+      // Overreid the default max operation count per batch.
+      const originalMaxOperationsPerBatch = ProtocolParameters.maxOperationsPerBatch;
+      ProtocolParameters.maxOperationsPerBatch = 1;
+
+      await expectAsync(AnchorFile.parseAndValidate(anchorFileCompressed))
         .toBeRejectedWith(new SidetreeError(ErrorCode.AnchorFileExceededMaxOperationCount));
+
+      ProtocolParameters.maxOperationsPerBatch = originalMaxOperationsPerBatch;
```

Comment:
Sanity check: should this be in 'try-finally'? Not sure if the expectAsync above fails then whether this line is executed or not.

Reply:
Thanks, good call!